index.md

```diff
@@ -2,131 +2,201 @@
+
+```console
+$ pwd
+/users/seshrs
+$ wget https://eecs485staff.github.io/primer-spec/demo/starter_files.tar.gz
+$ tar -xvzf starter_files.tar.gz
 ```
 
-#### Header 4
+### Preview the site on your browser
+Have a look at `index.html` by opening the file in your favorite text editor. This file holds all the HTML that represents this project spec. Notice how we've already included some CSS to make the spec look like GitHub's Primer theme.
 
-*   This is an unordered list following a header.
-*   This is an unordered list following a header.
-*   This is an unordered list following a header.
+Use Python's built-in static fileserver to view this HTML file on your browser. Open a console at the directory of the starter files, and then run the Python server.
 
-##### Header 5
+```console
+$ pwd
+/users/seshrs/primer-spec-spec
+$ python3 -m http.server
+```
 
-1.  This is an ordered list following a header.
-2.  This is an ordered list following a header.
-3.  This is an ordered list following a header.
+Visit [http://localhost:8000](http://localhost:8000) on your web browser. The page should look similar to this screenshot:
 
-###### Header 6
+<img src="https://gdurl.com/1Kq10" height="250" alt="project page without sidebar" />
 
-| head1        | head two          | three |
-|:-------------|:------------------|:------|
-| ok           | good swedish fish | nice  |
-| out of stock | good and plenty   | nice  |
-| ok           | good `oreos`      | hmm   |
-| ok           | good `zoute` drop | yumm  |
+The spec already looks pretty good, but it could certainly be improved with a sidebar.
 
-### This heading appears in the sidebar
+# HTML Sidebar
+In this section, you will add a sidebar to the HTML page and hard-code its contents. Although we'll be writing some JavaScript code later in the project to generate the contents of the sidebar, it's still a good idea to decide how it's going to look.
 
-Spam spam spam.
+When you're done with this section, your webpage will have a sidebar on the left, something like this screenshot:
 
-### This heading does NOT appear in the sidebar.
-{: .primer-spec-toc-ignore }
+<img src="https://gdurl.com/zJi1" height="250" alt="project page with sidebar" />
 
-(Add `{: .primer-spec-toc-ignore }` underneath the heading to prevent it from appearing in the sidebar.) Spam spam spam.
+Of course, _your_ finished webpage doesn't have to look like this. After all, this project isn't autograded! (In fact, feel free to showcase your project spec design with us! Create an issue on [our GitHub repository](https://github.com/eecs485staff/primer-spec/issues) with a screenshot of your design.)
 
-### There's a horizontal rule below this.
+However, if you wish to emulate our design, here are some tips.
 
-* * *
+#### Where do I put the sidebar code?
+We added our sidebar HTML at the very top of the `body` tag, just before the the main content begins. (The main content is contained in a `div` with class `markdown-body`.)
```

Comment:
Is the idea that I will write primer-spec from scratch?  Or should I copy-paste code from the repo at this point?

Reply:
I personally have been viewing the project as more of a gimmick, and don't really expect anyone to actually work on it (unless some dedicated 485 students discover it 😆).

When writing this part, I expected a dedicated reader to implement this on their own from scratch, while drawing ideas for implementation from the current behavior of Primer Spec.